rtv/multireddits.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
```

Comment:
True, these files are nearly identical. I was thinking perhaps have a separate file for different kinds of reddit pages might be easier to manage. Your alternative works too.

How would it work to use the same keybindings (`SUBSCRIPTION_SELECT`) to select two different pages? Do you mean have all subscribed subreddits and multireddits be listed together on one page?